src/components/FormModal.js

```diff
@@ -67,7 +67,7 @@ export default function FormModal({ open, handleClose, mode, selectedCar, modalK
 			setErrors({});
 			setForm(selectedCar);
 		} else setForm(initial_form);
-	}, [selectedCar?._id]);
+	}, [selectedCar]);
 	console.log('render');
```

Comment:
Delete this line.

Reply:
i did describe briefly why and what i did in the pull request description, but missing screenshots. I'll find a way to include them